ui-tests/src/test/resources/features/integrations/sql-connector.feature

```diff
@@ -0,0 +1,73 @@
+# @sustainer: jsafarik@redhat.com
+
+@ui
+@database
+@log
+@sql-connector
+Feature: SQL Connector
+
+  Background: Clean application state
+    Given clean application state
+    And log into the Syndesis
+    And reset content of "todo" table
+    And navigate to the "Home" page
+  #
+  # 1. Check that SQL connector pulls all rows on exchange when SQL connector is as middle step
+  # (having SQL connector as 'from' or 'to' doesn't cause issue from GH-3856)
+  #
+  @reproducer
+  @gh-3856
+  @sql-connector-all-rows
+  Scenario: Check SQL pulls all rows
+
+    Then inserts into "todo" table
+      | ABCsampleTODO |
+      | XYZsampleTODO |
+      | KLMsampleTODO |
+
+    When click on the "Create Integration" button to create a new integration
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    # select Timer as 'from' point
+    When select the "Timer" connection
+    And select "Simple Timer" integration action
+    And fill in values by element ID
+      | period        | 1       |
+      | select-period | Minutes |
+    And click on the "Done" button
+    Then check visibility of page "Choose a Finish Connection"
+
+    # select Log as 'to' point
+    When select the "Log" connection
+    And select "Simple Logger" integration action
+    And fill in values
+      | log level      | INFO  |
+      | Log Body       | true  |
+      | Log message Id | false |
+      | Log Headers    | false |
+      | Log everything | false |
+    Then click on the "Done" button
+
+    # select postgresDB as middle point
+    When add first step between START and STEP connection
```

Comment:
This looks like legacy step, please replace with  `add integration step on position "0"`

Reply:
updated